plugins/scaffolder-common/api-report.md

```diff
@@ -0,0 +1,36 @@
+## API Report File for "@backstage/plugin-scaffolder-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Entity } from '@backstage/catalog-model';
+import { JsonObject } from '@backstage/config';
+import { JSONSchema } from '@backstage/catalog-model';
+
+// @public (undocumented)
+export interface TemplateEntityV1beta3 extends Entity {
+  // (undocumented)
+  apiVersion: 'scaffolder.backstage.io/v1beta3';
+  // (undocumented)
+  kind: 'Template';
+  // (undocumented)
+  spec: {
+    type: string;
+    parameters?: JsonObject | JsonObject[];
+    steps: Array<{
+      id?: string;
+      name?: string;
+      action: string;
+      input?: JsonObject;
+      if?: string | boolean;
+    }>;
+    output?: {
+      [name: string]: string;
+    };
+    owner?: string;
+  };
+}
+
+// @public (undocumented)
+export const templateEntityV1beta3Schema: JSONSchema;
```

Comment:
you know actually, i wonder if it might have made sense to even let the type here be the full dumped schema contents (which means that completion and inspection would see into it)